core/lib/dal/src/via_votes_dal.rs

```diff
@@ -165,4 +165,82 @@ impl ViaVotesDal<'_, '_> {
 
         Ok(row.max_batch_number.map(|n| n as u32))
     }
+
+    pub async fn verify_votable_transaction(
```

Comment:
what's the difference between these two columns? finilized & verified? could you explain, please?

Reply:
nvm, got it